backend/spec/helpers/admin/navigation_helper_spec.rb

```diff
@@ -69,5 +69,10 @@
         end
       end
     end
+
+    it "should accept a block of content to append" do
+      admin_tab = helper.tab(:orders){ 'foo' }
```

Comment:
Looks like this file is a double-quoter. Would love to keep with that consistency.